tests/migration/online_migration/yast2_migration.pm

```diff
@@ -208,6 +209,13 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
+    # in leap to sle migration, we need click trust for package-hub's import-untrusted-gpg-key
+    if (is_leap_migration) {
+        assert_screen 'import-untrusted-gpg-key', 60;
+        if (match_has_tag('import-untrusted-gpg-key')) {
```

Comment:
This part, check tag two times. I think we can use one time.

Reply:
changed.